kratos/linear_solvers/linear_solver.h

```diff
@@ -273,6 +273,12 @@ class LinearSolver
         return 0;
     }
 
+    virtual unsigned int GetIterationsNumber()
+    {
+        KRATOS_WARNING("LinearSolver") << "WARNING: Accessed base function Kratos::LinearSolver::GetTolerance(), returning 0 !" << std::endl;
```

Comment:
```suggestion
        KRATOS_WARNING("LinearSolver") << "Accessed base function LinearSolver::GetTolerance, returning 0 !" << std::endl;
```

kratos warning already prints "warning"

Reply:
Done.